gwdetchar/lasso/__main__.py

```diff
@@ -344,7 +364,7 @@ def create_parser():
         '--remove-outliers-pf',
         type=float,
         default=None,
-        help='Percent limit for removing outliers',
+        help='Percent limit for removing outliers between 0 and 1',
```

Comment:
Instead of "Percent" suggest using "Fractional"

Reply:
Also done!